### DISCUSSION TITLE
Los tokens de mi billetera desaparecieron, no figura el movimiento

### DISCUSSION THREAD

**Discussion by JALL1962** [iotaledger/firefly#8182] [Help & Support]
Cuando me llegó el NFT de iota lottery, intenté hacer un renvio a la dirección de recibir fondos, en el momento del envío se fueron los tokens.
En Tangle Pay, aparece el movimiento; si alguien me puede ayudar para revertir el movimiento, me ayudaría mucho.
Como no soy desarrollador me pierdo.
Muchas gracias de antemano.
Saludos Jon Ander

**Comment by begonaalvarezd** [comment]
Hola, si has metido tu clave privada (eg tu mnemonic) en la web de iota lottery me temo que has sido victima de una estafa y te han robado. No podemos evitar que alguien te regale un NFT, las direcciones son públicas, por lo que cualquiera puede elegir una dirección y hacer un regalo; desafortunadamente, en este caso es un NFT con un mensaje de una web de estafa que se hace pasar por una wallet cuyo el unico objetivo es que le des tus claves y así poder robarte los fondos.

El objetivo de las wallets de autocustodia en crypto como Firefly/Bloom/TanglePay es que eres tu propio banco, por lo que tienes que cuidar muy bien tus claves porque eres el único que las posee. El ecosistema está lleno de advertencias como "no compartas tus claves con nadie, no ingreses tus claves en ninguna aplicación aleatoria...". 

Si has sido victima, sólo podemos sugerirte que lo informes a tus autoridades locales.

**Comment by JALL1962** [comment]
Hola Begoña,

Gracias por la explicación, me puedes decir qué debo hacer con el NFT que me enviaron y que está ubicado en la billetera?
Entiendo que el primer paso será cambiar la clave; tengo 2 preguntas:

- Tengo 2 preguntas:

Por qué en Firefly no aparece el apunte y en tangle pay si?

Esto se produjo, justo después de una actualización, por qué se ubicaron directamente en apartado de la cuenta de NFT?

Muchísimas gracias, Jon Ander

Enviado desde Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by begonaalvarezd** [reply]
> Por qué en Firefly no aparece el apunte y en tangle pay si?
Firefly no tiene todas las tx, si necesitas mirar todas tus txs puedes usar el explorer https://explorer.iota.org/
> Esto se produjo, justo después de una actualización, por qué se ubicaron directamente en apartado de la cuenta de NFT?
Con la actualizacion de la red se introdujo el soporte a NFTs que antes no habia, y con ello firefly tambien tenia que dar soporte

**Comment by georgeparker88** [comment]
Well that kind of sucks... Anybody reported these guys yet? Anyway to see to which address the funds have been transferred?

**Comment by georgeparker88** [comment]
The most stupid thing about this is that I thought it was like some of the previous messages we used to get about Shimmer and Assembly and came just after the update of the Wallet. Was loyal to the IOTA cause but I'm afraid I probably won't re-invest.

**Comment by JALL1962** [comment]
Gracias Begoña,

De cara a presentar una denuncia, me puedes asesorar sobre la información que debo aportar para que pueden saber la ruta que ha seguido el robo?
Piensa que no soy desarrollador, necesito paso a paso.

Saludos Jon Ander

Enviado desde Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
